pandas/core/internals/concat.py

```diff
@@ -323,7 +323,7 @@ def _concatenate_join_units(join_units, concat_axis, copy):
         # concatting with at least one EA means we are concatting a single column
```

Comment:
do we have any tests that hit this with concat_axis=1

Reply:
test_concat_frame_axis0_extension_dtypes goes through this path with concat_axis==1 despite the test name